### PR TITLE
3d world reconstruction

### PR DESCRIPTION
**Pull request by h3x4g0ns**
- created `lib/world.py` that will house tranformation tree to convert from im -> camera -> world which will allow us to ascertain the pose of the feline in the real world

### CODE REVIEW

File: lib/world.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  ax = fig.add_subplot(111, projection='3d')
+  ax.scatter(pts[:, 0], pts[:, 1], pts[:, 2], s=1)
+  ax.set_xlabel('X Coordinate')
+  ax.set_ylabel('Y Coordinate')
+  ax.set_zlabel('Z Coordinate')
+  plt.show()
+
+  # TODO:
+  # use pyvista or vtk for proper scene rendering
+
```
Comment (h3x4g0ns) [inline lib/world.py:86]: @AnirudhChaudhary need your help to come up with a better function for 3d voxel rendering so we can make sure that our transformations are accurate, pyvista and open3d seem the most promising, whichever is the easiest works

Comment (h3x4g0ns) [reply lib/world.py:86]: https://github.com/nerfstudio-project/viser

**Comment by h3x4g0ns** [comment]
World is currently at 0 translation and no rotation

**Review by AnirudhChaudhary — APPROVED** [review]
love the fact that the extrinisic matrix is simplified now. Let's hope that it works.

**Review by JayHil — APPROVED** [review]
wow, fantastic (I cannot read)

**Pull request merged**